tools/bitflow.py

```diff
@@ -0,0 +1,23 @@
+from crewai_tools import BaseTool
+from .bun import BunScriptRunner
+
+
+class ExecuteBitflowTradeTool(BaseTool):
+    """Tool for fetching contract code directly using user input."""
```

Comment:
Can you update this comment as well? Not sure if its pulled into anything with BaseTool vs tool decorator but nice to have the consistency.